providers/standard/tests/unit/standard/sensors/test_python.py

```diff
@@ -24,10 +24,15 @@
 
 from airflow.exceptions import AirflowSensorTimeout
 from airflow.providers.standard.sensors.python import PythonSensor
-from airflow.sensors.base import PokeReturnValue
 
+from tests_common.test_utils.version_compat import AIRFLOW_V_3_1_PLUS
 from unit.standard.operators.test_python import BasePythonTest
 
+if AIRFLOW_V_3_1_PLUS:
+    from airflow.sdk.bases.sensor import PokeReturnValue
+else:
+    from airflow.sensors.base import PokeReturnValue  # type: ignore[no-redef]
```

Comment:
You could just add this in version_compat.py and import them like:
```
from tests_common.test_utils.version_compat import PokeReturnValue
```

Reply:
I almost never recommend if / else anymore (might have done in the past but not anymore).

We have a couple of those occurrences in the repo which need cleanup, but trying to not introduce new ones. Because we compat here it between sdk / providers, not core. The `AIRFLOW_V_x_y_PLUS` checks compatibility based on version of airflow core which semantically isn't correct even, so while we can, better to have try / except (more robust too)